static/js/AdminEditor.jsx

```diff
@@ -213,6 +213,8 @@ const AdminEditor = ({title, data, close, catMenu, pictureUploader, updateData,
                             return catMenu;
                         } else if (x === "Picture Uploader") {
                             return pictureUploader;
+                        } else if (x === "Secondary Picture Cropper") {
```

Comment:
maybe it's better to replace these in-render "if" statements with a map data-structure to map items to the appropriate elements to render?